lib/ops/ops.go

```diff
@@ -855,6 +855,143 @@ type Operations interface {
 	GetOperationPlan(SiteOperationKey) (*storage.OperationPlan, error)
 }
 
+// OperationsFilter represents a filter to apply to results when listing operations
+type OperationsFilter struct {
+	// Last indicates to only return the last operation
+	Last bool
+
+	// First indicates to only return the first operation
+	First bool
+
+	// Complete indicates to only return completed operations
+	Complete bool
+
+	// Finished indicates to only return finished operations (complete or failed)
+	Finished bool
+
+	// Active indicate to only return active operations
+	Active bool
+
+	// Type indicates to only return a type of operation
```

Comment:
The comment is a little confusing - do you mean only return a particular type of operation? If so, maybe also make this a slice to allow to return multiple types of operations?